CURLE_SSL_CACERT is defined as CURLE_SSL_PEER_FAILED_VERIFICATION since curl-7.62

--- daemon/http-tx-mgr.c.orig  2018-11-01 18:51:10.712479238 +0100
+++ daemon/http-tx-mgr.c       2018-11-01 18:52:36.115684514 +0100
@@ -1136,7 +1136,6 @@
     case CURLE_SSL_CONNECT_ERROR:
     case CURLE_PEER_FAILED_VERIFICATION:
     case CURLE_SSL_CERTPROBLEM:
-    case CURLE_SSL_CACERT:
     case CURLE_SSL_CACERT_BADFILE:
     case CURLE_SSL_ISSUER_ERROR:
         return HTTP_TASK_ERR_SSL;
